src/virtual-select.js

```diff
@@ -448,6 +448,7 @@ export class VirtualSelect {
     this.$toggleAllCheckbox = this.$dropboxContainer.querySelector('.vscomp-toggle-all-checkbox');
 
     this.addEvent(this.$searchInput, 'input', 'onSearch');
+    this.addEvent(this.$searchInput, 'change', 'toPreventEvent');
```

Comment:
The method name `toPreventEvent` is ambiguous; consider renaming to `preventChangePropagation` for clarity.